src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -354,14 +354,15 @@ static StringId window_cheats_page_titles[] = {
     STR_CHEAT_TITLE_PARK,
     STR_CHEAT_TITLE_RIDE,
 };
+
+static int32_t window_cheats_selectedStaffSpeed = 1;
```

Comment:
This should be moved below the clang format on line

Reply:
Addressed in newest amend.